src/excmds.ts

```diff
@@ -751,6 +751,26 @@ export async function source_quiet(...fileArr: string[]) {
     }
 }
 
+/** Use tridactylrc located at the given url
+ * @param url the url where the raw version of tridactylrc can be found
```

Comment:
Thanks for this PR! I didn't realise it would be so easy. Could you check that `autocmd TriStart .* source-from_url ...` works, and if it does, add it to this help?

Reply:
How do I test this? `src/lib/config.ts` has a default `autocmd` on `TriStart` but replacing it `".*": "source_from_url_quiet(fixed_url)"` doesn't do anything.

Reply:
That should work. I guess it doesn't, so don't add it to the help :)